environment/service.go

```diff
@@ -30,20 +30,20 @@ var (
 	VersionFabric8TenantJenkinsQuotasFile string
 	VersionFabric8TenantCheQuotasFile     string
 	VersionFabric8TenantDeployFile        string
-	DefaultEnvTypes                       = []string{"che", "jenkins", "user", "run", "stage"}
+	DefaultEnvTypes                       = []Type{TypeChe, TypeJenkins, TypeRun, TypeStage, TypeUser}
```

Comment:
I guess it doesn't matter but you are changing the order of types here. Just wanted to bring your attention in case we actually relay on the order somewhere. If not then it's fine.

Reply:
I'm aware of it and it doesn't influence anything - nothing in the code relies on the order of the values in this slice